app/next.config.js

```diff
@@ -1,6 +0,0 @@
-/** @type {import('next').NextConfig} */
```

Comment:
For this file and the next I'm just experiencing some git weirdness. I moved the app to `next-app` to disambiguate it from `vite-app` but something obviously went wrong in the process.